src/Backend/OnboardingInsuranceAPI/Areas/Insurance/ContributionData.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace OnboardingInsuranceAPI.Areas.Insurance;
+
+public record ContributionData
+{
+    [JsonPropertyName("expDate")]
+    public string ExpirationDate { get; set; }
```

Comment:
Why are some set and some init? You are using records. This is confusing. Either use "with" and create new instance of the record. This doesn't respect immutability.

Reply:
Yeah, thanks for explanation. I thought I can use same data properties for request and response. I'll divide it.